Enable artifacts as default (#204)
kubeflow/kfp-tekton: tekton_kfp_guide.md

```

### Backend

> The KFP backend with Tekton needs to modify the Kubeflow Pipelines api-server, persistent agent, and metadata writer. 

Reviewer: ```suggestion
The KFP backend with Tekton uses a modified version of Kubeflow Pipelines api-server, persistent agent, and metadata writer. 
```
